backend/src/core/services/campaign.service.ts

```diff
@@ -76,6 +76,17 @@ const listCampaigns = ({
           ['updated_at', 'status_updated_at'],
         ],
       },
+      {
+        model: Statistic,
+        attributes: [
+          [
+            literal(
+              'CASE WHEN (errored + invalid) > 0 THEN True ELSE False END'
```

Comment:
```suggestion
              '(errored + invalid) > 0'
```

Reply:
can u also change the literal above to ```cred_name IS NOT NULL```

Reply:
This returns `{ statistics: null }` when the campaign is not logged, so I did this to conform it to `{ statistics: 
{ has_failed_recipients: true } }`, do you think returning null is clearer for a campaign which is not logged?

Reply:
null case has been handled by this right?
```
getFailedRecipients(statistic: { has_failed_recipients: boolean }): boolean {
    return statistic && statistic.has_failed_recipients
  }
```

Reply:
yeah true, okay will replace the literal